packages/trip-details/src/index.tsx

```diff
@@ -291,12 +292,14 @@ export function TripDetails({
         {fare && (
           <TripDetail
             // Any custom description for the transit fare needs to be handled by the slot.
+            //  TODO: add leg info
             description={
-              FareDetails && (
+              fareDetailsLayout &&
+              Object.keys(itinerary.fare.details).length > 0 && (
                 <FareDetails
-                  maxTNCFare={maxTNCFare}
-                  minTNCFare={minTNCFare}
-                  transitFares={transitFares}
+                  transitFares={itinerary.fare}
+                  legs={itinerary.legs}
```

Comment:
Sort props

Reply:
This could be cool, but I think the table is formatted to spec based on an outside request

Reply:
Basically, there is duplication of information, so to me it makes sense to at least remove the exiting fare list ((keep the default fare total) in favor of the new fare table. If we wanted we could have a new condensed state in the fare table that would look like:

### Transit Fare: $5.50

| Adult | Cash<br>$5.50 | Orca<br>$3.25 | Orca Lift<br>$1.50 |
|-------|-------|-------|--------|
| Youth | Cash<br>$3.00 | Orca<br>$1.50 |       |
| Senior | Cash<br>$2.00 | Orca<br>$1.00 |       |

[Expand]

Reply:
I see what you mean but I'm still in favor of both as they provide different utility. However, I'll defer to @daniel-heppner-ibigroup as the designer of this! I see your argument as well

Reply:
This is a tough one! I like the idea of reducing duplication, but I find the big table to be a bit confusing. I think when you're looking for your fare type it's easier to just look at a list rather than find two axes on a table. 

As Miles said, the other problem here is that the layout of the fare by leg information is very specific based on an outside request.